Makefile.am

```diff
@@ -51,7 +51,7 @@ test_extra_programs = \
 	tests/try-syscall \
 	$(NULL)
 
-test-bwrap: bwrap
+test-bwrap$(EXEEXT): bwrap
```

Comment:
This part looks fine, but please make it its own commit. If you find yourself writing out a list in a commit message, that's often a sign that it should have been more than one commit.